taxi/views.py

```diff
@@ -36,6 +43,23 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs) -> dict:
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
```

Comment:
here you can omit passing class explicitly in super() - it is only required in python 2